tests/test_crypto_utils.c

```diff
@@ -10,6 +10,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+
+#include "unity.h"
+
```

Comment:
Ditto.

Reply:
Done.
I reorganize the alphabetical order of the headers as well.